server/app/views/components/TextFormatter.java

```diff
@@ -50,6 +51,8 @@ public enum UrlOpenAction {
   private static final String ACCORDION_CONTENT = ">";
   private static final String ACCORDION_HEADER = "### ";
```

Comment:
Suggestion (non-blocking): In the static questions, I think it will be confusing that we're using these markdown symbols to define the accordion rather than to create a blockquote ('>') or an H3('###').  I suggest using non-markdown characters to delineate the accordion.  This could be a fast-follow. 
IIRC, there were no static questions in the Seattle db when @gwendolyngoetz last checked.  If that's the case for other civic entities, then we wouldn't have to worry about backward compatibility and could just switch out the symbols.

Reply:
When I check last there were no uses of accordion in Seattle. The only one I can think of specifically is on the api docs page.

My suggestion for new characters:
`ACCORDION_CONTENT="🪗";`
`ACCORDION_HEADER = "🎩";`

Reply:
Lol those characters!! I agree this will be confusing, and would actually prevent users from using H3s. I would rather do it as a fast follow so that we can quickly revert in case it messes up existing values. I'll make an issue for it.